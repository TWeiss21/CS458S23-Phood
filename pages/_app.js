import '@/styles/globals.css'
import "../styles/dashboard.css";
import "../styles/header.css";
import "../styles/recipeList.css";
import "../styles/recipeDetails.css";
import "../styles/shoppingList.css";
import "../styles/AddRecipeModal.css";
import "../styles/NewApiModal.css";
import "../styles/ExportModal.css";

export default function App({ Component, pageProps }) {
  return <Component {...pageProps} />
}
