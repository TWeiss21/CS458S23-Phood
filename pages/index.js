import Dashboard from '../components/dashboard.jsx'


const Index = () => (
    <div>
        <Dashboard/>
    </div>
)

export default Index