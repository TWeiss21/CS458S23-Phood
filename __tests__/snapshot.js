import { Dashboard } from '@mui/icons-material'
import { render } from '@testing-library/react'
import { Index, Home } from '../pages/index'

it("renders homepage unchanged", () => {
    const { container } = render(<Dashboard/>);
    expect(container).toMatchSnapshot();
});
