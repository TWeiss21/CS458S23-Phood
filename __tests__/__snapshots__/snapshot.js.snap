// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <div
      class="layout"
    >
      <div
        class="header"
      >
        <div>
          <div
            class="headerDiv"
          >
            <button
              class="btn"
            >
              Dashboard
            </button>
          </div>
          <div
            class="headerDiv SL"
          >
            <button
              class="btn"
            >
              Shopping List
            </button>
          </div>
        </div>
      </div>
      <div
        class="recipeList centered"
      >
        <div>
          <div
            class="recipeListHeader"
            id="recipeListHeader"
          >
             ‚Äç Recipe Book
          </div>
          <div
            class="listContainer"
            id="listContainer"
          >
            <div
              class="outerRecipeContainer"
              id="outerRecipeContainer"
            >
              <button
                class="recipeContainer"
                id="recipeContainer"
              >
                <div
                  class="recipeListName"
                  id="recipeListName"
                >
                  Chicken Nuggets
                </div>
                <div
                  class="recipeListLine"
                  id="recipeListLine"
                />
              </button>
              <button
                class="listPlus"
                id="listPlus"
              >
                +
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="recipeDetail centered"
      >
        <div>
          RecipeDetail
        </div>
      </div>
      <div
        class="pantry centered"
      >
        <div>
          Pantry
        </div>
      </div>
    </div>
  </div>
</div>
`;
